jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>can you maybe motiviate this section a little bit further? what are the insight you can generate apart from inferring the composition?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>add a sentence about the biological results we get</p>

Reply:
<p>Okay now?</p>

Reply:
<p>I would also add more context for bulk sequencing here. Don't think that you ever mention that a fundamental limitation of bulk sequencing is that it averages expression and therefore cell types cannot be properly distinguished etc.</p><p> </p><p>Problem first, then opportunities (you have this) and then solution (you have this)</p>

Reply:
<p>Why is that related to cell deconvolution? This is the advantage of single cell RNA-seq over RNA-seq but it's not something that deconvolution solve... Maybe I didn't understand your point.</p>